tests/01_eradiate/01_unit/scenes/surface/test_dem.py

```diff
@@ -127,21 +127,39 @@ def make_dataarray(data: pint.Quantity, coords: dict):
             },
             [(-0.089832, 0.179664), (-0.269496, 0.359328)],
         ),
+        (
+            {
+                "da": make_dataarray(
+                    data=np.zeros((10, 10)) * ureg.m,
+                    coords={
+                        "x": np.linspace(-10, 20, 10) * ureg.km,
+                        "y": np.linspace(-30, 40, 10) * ureg.km,
+                    },
+                ),
+                "geometry": "dodecahedron",
+                "planet_radius": EARTH_RADIUS,
+            },
+            [(-0.089832, 0.179664), (-0.269496, 0.359328)],
+        ),
     ],
 )
 def test_mesh_from_dem(modes_all_double, kwargs, expected_limits):
-    if (kwargs["geometry"] == "spherical_shell") and kwargs[
-        "planet_radius"
-    ] is not None:
-        with pytest.warns():
+    if kwargs["geometry"] not in ["spherical_shell", "plane_parallel"]:
+        with pytest.raises(ValueError):
             mesh, lat, lon = mesh_from_dem(**kwargs)
```

Comment:
Assigning is not necessary:
```suggestion
            mesh_from_dem(**kwargs)
```